packages/desktop-client/src/components/modals/AccountAutocompleteModal.tsx

```diff
@@ -13,13 +15,14 @@ import { View } from '../common/View';
 import { SectionLabel } from '../forms';
 import { useResponsive } from '../responsive/ResponsiveProvider';
 
-type AccountAutocompleteModalProps = {
-  autocompleteProps: ComponentPropsWithoutRef<typeof AccountAutocomplete>;
-  onClose: () => void;
-};
+type AccountAutocompleteModalProps = Extract<
```

Comment:
Updated modals to re-use the options type defined in `modalsSlice` so that we don't have to define modal props/options in 2 places